### ISSUE TITLE
`cargo install elba` fails on rust/cargo 1.34

### ISSUE THREAD

**Issue by stephen-smith** [elba/elba#80]
Probably just want to update the docs to indicate a newer version is required, but my Debian rust 1.34 installation wasn't happy installing and current docs say 1.31 is sufficient:

```
error: aborting due to 23 previous errors

Some errors occurred: E0599, E0658.
For more information about an error, try `rustc --explain E0599`.
error: Could not compile `smallvec`.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `elba v0.3.0`, intermediate artifacts can be found at `/tmp/cargo-installdAsgLu`

Caused by:
  build failed
bss@monster Exit Code: 101 % rustc --version
rustc 1.34.2
bss@monster % cargo --version
cargo 1.34.0
```

**Comment by stephen-smith** [comment]
Just an FYI for anyone else seeing this, upgrading (just) rustc/cargo from testing/sid doesn't help, it actually just breaks your cargo installation:

```
bss@monster % rustc --version
rustc 1.42.0
bss@monster % cargo --version
cargo 1.42.1
bss@monster % cargo install elba
    Updating crates.io index
error: failed to fetch `https://github.com/rust-lang/crates.io-index`

Caused by:
  the SSL certificate is invalid: 0x08 - The certificate is not correctly signed by the trusted CA; class=Ssl (16); code=Certificate (-17)
```

(This isn't entirely unexpected, I'm probably mixing ssl/tls libraries from stable with stuff from testing, which is not supported at all, and just "normally" works for me.  Like I said, just a FYI if you stumble along behind me.)

**Comment by andylokandy** [comment]
The elba on crates.io is extremely outdated, so I recommend to use the latest nightly rustc and build from source.

**Comment by stephen-smith** [comment]
The release binary from here (Gtihub) worked for me.  Still, it would be nice for the README.md to reflect the current state, even if that is currently "grab everything from HEAD and good luck". ;)

**Comment by andylokandy** [comment]
Thanks! I updated the readme doc and released a new version of 0.3.3.
